components/Cards/SchoolCard/__tests__/SchoolCard.test.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import createShallowSnapshotTest from 'test-utils/createShallowSnapshotTest';
+import SchoolCard from '../SchoolCard';
+
+const locations = [
+  {
+    va_accepted: true,
+    address1: '825 Battery Street',
+    address2: '3rd Floor',
+    city: 'San Francisco',
+    zip: '94111',
+  },
+];
+
+describe('SchoolCard', () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = shallow(
+      <SchoolCard
+        cardFlipCallback={() => {}}
+        hasHardwareIncluded
+        hasHousing
+        hasOnline
+        hasOnlyOnline={false}
+        isFullTime
+        locations={locations}
+        logoSource="source"
+        name="school name"
+        website="website"
+      >
+        Test
+      </SchoolCard>,
+    );
+  });
+
+  it('should render with required props', () => {
+    createShallowSnapshotTest(
+      <SchoolCard
+        hasHardwareIncluded
+        hasHousing
+        hasOnline
+        hasOnlyOnline={false}
+        isFullTime
+        locations={locations}
+        logoSource="logoSource.com"
+        name="school name"
+        website="website.com"
+      >
+        <p>Testing!</p>
+      </SchoolCard>,
+    );
+  });
+
+  it('call to showBackOfCard/showFrontOfTheCard flips isFrontOfCardShowing', () => {
+    const instance = wrapper.instance();
+    expect(wrapper.state('isFrontOfCardShowing')).toBe(true);
+    instance.showBackOfCard();
+    expect(wrapper.state('isFrontOfCardShowing')).toBe(false);
+    instance.showFrontOfCard();
+    expect(wrapper.state('isFrontOfCardShowing')).toBe(true);
+  });
+
+  it('renders the See Locations Buttom', () => {
+    expect(wrapper.state('isFrontOfCardShowing')).toBe(true);
+    expect(wrapper.find('See Locations')).toBeTruthy();
+  });
```

Comment:
Good idea to test this the existence of the "See Locations" button. 

We always want to think about "what are we really testing?"

Looking at the logic of the component, this button will only render if there are multiple locations passed to the component. Let's test that!

```javascript
it('should render the "See Locations" button when multiple locations exist', () => {
  // ensure wrapper has multiple locations at this point
  expect(wrapper.prop('locations')).toBeGreaterThan(1);
  expect(wrapper.find('button').filterWhere(node => node.text() === 'See Locations')).toExist();
});
```